modules/ssh-agent-using.adoc

```diff
@@ -180,6 +195,20 @@ Identity added: /home/<you>/<path>/<file_name> (<computer_name>)
 ----
 <1> Specify the path and file name for your SSH private key, such as `~/.ssh/id_rsa`
 
+ifdef::gcp[]
+. Set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable to the full path to your service account private key file.
++
+[source,terminal]
+----
+$ export GOOGLE_APPLICATION_CREDENTIALS="<your_service_account_file>"
+----
+.. Verify that the credentials were applied.
```

Comment:
Does it make sense to have this as a second level step? `..` vs `.`

Reply:
I was torn honestly. I'll just make it single level for consistency so it isn't an oddball.